### PR TITLE
Add localization messages for 'fi' locale

### PR DESCRIPTION
**Pull request by BingoRUS**
Finnish localization messages are provided by @aenges.

### CODE REVIEW

File: js/localization/messages/fi.json [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+
+        "dxNumberBox-invalidValueMessage": "Arvon on oltava numero",
+
+        "dxDataGrid-columnChooserTitle": "Sarakkeenvalitsin",
+        "dxDataGrid-columnChooserEmptyText": "Vedä sarake täällä piilottaa sen",
+        "dxDataGrid-groupContinuesMessage": "Jatkuu seuraavalla sivulla",
+        "dxDataGrid-groupContinuedMessage": "Jatkoa edelliseltä sivulta",
+        "dxDataGrid-groupHeaderText": "Ryhmittele tämän sarakkeen mukaan",
+        "dxDataGrid-ungroupHeaderText": "Poista ryhmittely",
+        "dxDataGrid-ungroupAllText": "Poista ryhmittely",
```
Comment (BingoRUS) [inline js/localization/messages/fi.json]: The constants "dxDataGrid-ungroupHeaderText" and "dxDataGrid-ungroupAllText" are the same. Can we change that?

File: js/localization/messages/fi.json [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+        "dxDataGrid-ariaDataGrid": "Tietoruudukko",
+        "dxDataGrid-ariaSearchInGrid": "Hae tietoriviltä",
+        "dxDataGrid-ariaSelectAll": "Valitse kaikki",
+        "dxDataGrid-ariaSelectRow": "Valitse rivi",
+
+        "dxTreeList-ariaTreeList": "Puu",
+        "dxTreeList-editingAddRowToNode": "Lisää",
+
+        "dxPager-infoText": "Sivu {0}/{1} ({2} kohdetta)",
+        "dxPager-pagesCountText": "/",
```
Comment (BingoRUS) [inline js/localization/messages/fi.json:154]: Is there no equivalent for "of" in Finnish?

**Review by BingoRUS — COMMENTED** [review]
Hi @aenges!
Please take a look or my comments and let me know if it's possible to improve the translation.

File: js/localization/messages/fi.json (hunk 2 above)
Comment (aenges) [reply js/localization/messages/fi.json:154]: Not really. For example "three [out] of five" is "Kol­me vii­des­tä" where the suffix "stä" means of. But you cannot write 3 5stä, the suffix is only used on textual representations of numbers. I looked at how Microsoft had solved this and they just used / between the numbers.

Comment (BingoRUS) [reply js/localization/messages/fi.json:154]: Okay, thanks for clarification!

File: js/localization/messages/fi.json (hunk 1 above)
Comment (aenges) [reply js/localization/messages/fi.json]: We could use:
"dxDataGrid-ungroupAllText": "Poistaa kaikki ryhmittymiä",

Comment (aenges) [reply js/localization/messages/fi.json]: Sorry. My wife came home and wanted me to change to: 
"dxDataGrid-ungroupAllText": "Poista kaikki ryhmittymät"

Comment (BingoRUS) [reply js/localization/messages/fi.json]: Okay, I used the last variant.

**Review by San4es — APPROVED** [review]

**Pull request merged**